src/webapp/app/Client.Application/Layout/MainLayout.razor

```diff
@@ -19,9 +19,22 @@
             <div class="profile">
                 <AuthorizeView>
                     <Authorized>
-                        <FluentButton BackgroundColor="var(--neutral-layer-4)"  @onclick="BeginLogOut" Title="Profile">
-                            <FluentEmoji Value="@(new Emojis.AnimalsNature.Flat.Default.Lion())" Width="35px" />
+                        <FluentButton id="btnOpenProfile" BackgroundColor="var(--neutral-layer-4)"  @onclick="() => _visibleL = !_visibleL" Title="Profile">
+                            <FluentPersona Name="@context.User.Identity?.Name"

```

Comment:
Please check for null value effect on the frontend

Reply:
Added check to handle null